CHANGELOG.md

```diff
@@ -30,6 +31,68 @@
 - [0.10.0](#0100---20170307)
 - [0.9.9 and prior](#099---20170202)
 
+## [1.4.0rc1]
+
+> Released on 2019/09/xx
+
+### Additions
+
+##### Core
+
+  - New Admin API endpoint `/status` has been introduced. It exposes insensitive
```

Comment:
This is not an endpoint in the Admin API, this is a new status interface separate from the Admin API. (The `/status` endpoint in the Admin API is something else which already exists.)